src/Avalonia.Visuals/Animation/Animators/RelativePointAnimator.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Avalonia.Animation.Animators
+{
+    /// <summary>
+    /// Animator that handles <see cref="RelativePoint"/> properties.
+    /// </summary>
+    public class RelativePointAnimator : Animator<RelativePoint>
+    {
+        private static readonly PointAnimator s_pointAnimator = new PointAnimator();
+
+        public override RelativePoint Interpolate(double progress, RelativePoint oldValue, RelativePoint newValue)
+        {
+            if (oldValue.Unit != newValue.Unit)
+            {
+                return progress >= 1 ? newValue : oldValue;
```

Comment:
In other places we are usually show the old value until 0.5.

Reply:
yeah, it should change halfway rather than at the end @maxkatz6

Reply:
Done